bundle/edu.gemini.ags.servlet/src/main/java/edu/gemini/ags/servlet/ToContext.java

```diff
@@ -151,9 +151,10 @@ private TargetEnvironment targetEnv(HttpServletRequest req) throws RequestExcept
             // Here we just need an SPTarget with a contained NonSiderealTarget. In the end, this
             // is just used to determine whether it is a non-sidereal target or not in order to
             // pick the right guider.
+            // TODO: track down where this happens!
```

Comment:
I think it is probably just AGS.  It wants to know which strategy should be the default.  If it is a non-sidereal target usually a PWFS guide probe is used whereas sidereal targets use an OIWFS by default.

See [this code](https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/Strategy.scala#L65-L75) for example.